packages/codegen/templates/app00/bin/prepare.js

```diff
@@ -0,0 +1,35 @@
+const { prepareCondoAppOidcConfig, prepareCondoAppB2BAppConfig, updateAppEnvFile, prepareAppEnvLocalAdminUsers, getAppEnvValue, runAppPackageJsonScript } = require('@open-condo/cli')
+
+
+const main = async ([B2BAppName = '{{name}}']) => {
+    // 1) register oidc url in condo
+    const appName = '{{name}}'
+    const oidcConf = await prepareCondoAppOidcConfig(appName)
+    await updateAppEnvFile(appName, 'OIDC_CONDO_CLIENT_CONFIG', JSON.stringify(oidcConf))
+    await updateAppEnvFile(appName, 'APP_TOKEN_KEY', 'appToken')
+    await updateAppEnvFile(appName, 'CONDO_ACCESS_TOKEN_KEY', 'condoAccessToken')
+    await updateAppEnvFile(appName, 'CONDO_REFRESH_TOKEN_KEY', 'condoRefreshToken')
+    await updateAppEnvFile(appName, 'CONDO_ORGANIZATION_ID_KEY', 'condoOrganizationId')
+    await updateAppEnvFile(appName, 'ACCEPT_LANGUAGE', 'accept-language')
```

Comment:
Why write this into environment variables?

Reply:
To get access in `generate.condo.server.utils`